math/src/main/java/com/powsybl/math/matrix/SparseLUDecomposition.java

```diff
@@ -71,7 +84,11 @@ private void checkMatrixStructure() {
     @Override
     public void update() {
         checkMatrixStructure();
-        update(id, matrix.getColumnStart(), matrix.getRowIndices(), matrix.getValues(), matrix.getRgrowthThreshold());
+        Stopwatch stopwatch = Stopwatch.createStarted();
+        double rgrowth = update(id, matrix.getColumnStart(), matrix.getRowIndices(), matrix.getValues(), matrix.getRgrowthThreshold());
+        stopwatch.stop();
+        LOGGER.debug("Sparse LU decomposition updated (rgrowth is {}, threshold is {}) in {} us",
```

Comment:
Maybe to be clearer you can add that it is the rgrowth from the refactor: "Sparse LU decomposition updated (**refactor** rgrowth is {}"